src/js/utils.js

```diff
@@ -1,3 +1,4 @@
+import {merge} from 'lodash';
```

Comment:
I wanted to point you to lodash, but I thought you might have considered it already :)